src/NuGet.Clients/NuGet.VisualStudio.Client/source.extension.vsixmanifest

```diff
@@ -1,23 +1,23 @@
 <?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
     <Metadata>
-        <Identity Id="NuGet.72c5d240-f742-48d4-a0f1-7016671e405b" Version="16.99.99" Language="en-US" Publisher="Microsoft Corporation" />
-        <DisplayName>NuGet Package Manager for Visual Studio 2019</DisplayName>
+        <Identity Id="NuGet.72c5d240-f742-48d4-a0f1-7016671e405b" Version="17.99.99" Language="en-US" Publisher="Microsoft Corporation" />
+        <DisplayName>NuGet Package Manager for Visual Studio 2022</DisplayName>
         <Description>A collection of tools to automate the process of downloading, installing, upgrading, configuring, and removing packages from a VS Project.</Description>
         <PackageId>Microsoft.VisualStudio.NuGet.Core</PackageId>
         <License>eula.rtf</License>
         <AllowClientRole>true</AllowClientRole>
     </Metadata>
     <Prerequisites>
-        <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[16.0,17.0)" DisplayName="Visual Studio core editor" />
+        <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[17.0,18.0)" DisplayName="Visual Studio core editor" />
     </Prerequisites>
     <Installation SystemComponent="true" InstalledByMsi="false" AllUsers="true">
-        <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[16.0,17.0)" />
-        <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[16.0,17.0)" />
-        <InstallationTarget Id="Microsoft.VisualStudio.VWDExpress" Version="[16.0,17.0)" />
-        <InstallationTarget Id="Microsoft.VisualStudio.VPDExpress" Version="[16.0,17.0)" />
-        <InstallationTarget Id="Microsoft.VisualStudio.VSWinExpress" Version="[16.0,17.0)" />
-        <InstallationTarget Id="Microsoft.VisualStudio.VSWinDesktopExpress" Version="[16.0,17.0)" />
+        <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[17.0,18.0)">
+            <ProductArchitecture>amd64</ProductArchitecture>
+        </InstallationTarget>
+        <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[17.0,18.0)">
+            <ProductArchitecture>x86</ProductArchitecture>
```

Comment:
Are the `ProductArchitecture` tags needed? Since NuGet is AnyCPU, we shouldn't care about which architecture we're running on, as long as .NET itself works.  If we leave it out, like we had in the past, does it allow NuGet to be installed on any architecture?

Reply:
There is no supported equivalent of AnyCPU/neutral. The enum is [here](https://devdiv.visualstudio.com/DevDiv/_git/VSExtensibility?path=%2Fsrc%2Fproduct%2FextensionManager%2Fshared%2FExtensionEngine%2FDef%2FProductArchitecture.cs&_a=contents&version=GBdevelop), and the validation code is [here](https://devdiv.visualstudio.com/DevDiv/_git/VSExtensibility?path=%2Fsrc%2Fproduct%2Fvssdk%2Ftools%2FVsixUtil%2FValidator.ProductArchitecture.cs&_a=contents&version=GBdevelop).